counter-sink/src/main/java/org/springframework/cloud/stream/module/metrics/CounterSinkConfiguration.java

```diff
@@ -15,35 +15,16 @@
  */
 package org.springframework.cloud.stream.module.metrics;
 
-import javax.annotation.PostConstruct;
-
-import org.springframework.beans.factory.BeanFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.expression.EvaluationContext;
-import org.springframework.integration.context.IntegrationContextUtils;
 
 /**
+ * Configuration class for the `counter-sink`.
+ *
  * @author Mark Pollack
+ * @author Ilayaperumal Gopinathan
  */
 @Configuration
-@EnableConfigurationProperties(CounterSinkProperties.class)
+@EnableConfigurationProperties(MetricProperties.class)
 public class CounterSinkConfiguration {
```

Comment:
Seems like this class is empty after refactoring. Can the `@EnableConfigurationProperties` be moved to the CounterSink class itself ?